app/models/PointerHelper.scala

```diff
@@ -0,0 +1,163 @@
+/*
```

Comment:
Since `PointerHelper` is used by templates, couldn't it be placed in `views/helpers`?